package.json

```diff
@@ -46,10 +49,15 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-svelte3": "^3.2.1",
     "jest": "^27.4.5",
+    "mdsvex": "^0.9.8",
+    "nprogress": "^0.2.0",
```

Comment:
Is this disabled if [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) is configured? Generally the browser busy indicators handle page load status and that's what the user expects and has control over at the system level from an accessibility perspective. If this library is the one I remember, it was created before browsers all had native support for busy indicators on SPAs.